doc/source/rllib/rllib-training.rst

```diff
@@ -489,16 +489,16 @@ The following is a list of the common algorithm hyper-parameters:
     # If - after one `step_attempt()`, the time limit has not been reached,
     # will perform n more `step_attempt()` calls until this minimum time has been
     # consumed. Set to 0 for no minimum time.
-    "min_time_s_per_reporting": 0,
+    "min_time_s_per_iteration": 0,
```

Comment:
I want to double check, because I definitely remember we changed the name from min_time_s_per_iteration to min_time_s_per_reporting recently.
https://github.com/ray-project/ray/pull/24372
which is part of 1.13 release I think?
so we are asking people to migrate to the new name, and back to the old name in 2.0?
I want to see if this is really worth it, maybe there is no perfect name for these parameters ... :)

Reply:
this makes total sense.
all things considered, I do think per_iteration being a better name here.